Add Romy integration documentation (#27578)
home-assistant/home-assistant.github.io: source/_integrations/romy.markdown

---
title: ROMY
description: Integrate your ROMY vacuum robot with Home Assistant.
ha_category:
  - Vacuum
ha_iot_class: Local Polling

> ha_release: 2023.5

Reviewer: ```suggestion
ha_release: 2024.2
```

ha_config_flow: true
ha_codeowners:
  - '@xeniter'
ha_domain: romy
ha_platforms:

>   - binary_sensor
>   - sensor
>   - vacuum
>   - button

Reviewer: ```suggestion
  - vacuum
```

PR includes only the vacuum platform

ha_integration_type: integration
---

> The `romy` integration allows you to control your [ROMY](https://www.romyrobot.com) vacuum robot.

Reviewer 2: ```suggestion
The **ROMY** integration allows you to control your [ROMY](https://www.romyrobot.com) vacuum robot.
```
Tiny tweak. We spell the integration name the way it is shown in the UI.

> This integration supports currently following models:

Reviewer 3: ```suggestion
This integration currently supports the following models:
```

- ROMY C5
- ROMY L6 Performance
- ROMY L6 Animal

{% include integrations/config_flow.md %}

## Services

Currently supported services are:

- `start`
- `pause`
- `continue`
- `stop`
- `return_to_base`

> ## Sensors
> 
> Currently supported sensors are:
> 
> - battery level
> - wifi rssi level
> - dustbin level
> - docked
> - dustbin present
> - watertank present
> - watertank empty
> 

Reviewer: ```suggestion
```

## Troubleshooting

> ### local HTTP interface password

Reviewer 2: ```suggestion
### Local HTTP interface password
```

> You have to enable the local interface first with a password, this is printend as QR Code on a label directly under the dustbin inside the robot.

Reviewer 3: ```suggestion
You have to enable the local interface first with a password, this is printed as QR Code on a label directly under the dustbin inside the robot.
```

> You have to enable the local interface first with a password, this is printed as QR Code on a label directly under the dustbin inside the robot.

Reviewer 2: ```suggestion
You have to enable the local interface first with a password. This is printed as QR Code on a label directly under the dust bin inside the robot.
```

### ROMY robot interface protocol

> Is available as download under [romy-robot-interface-protocol](https://romyrobot.eu/pages/romy-robot-interface-protocol)

Reviewer 3: This link currently redirects to the ROMY homepage. Has the download moved?

Author: homepage url has changed, corrected it now
